tests/controls/webtest.cpp

```diff
@@ -55,9 +74,22 @@ class WebTestCase : public CppUnit::TestCase
     void Editable();
     void Selection();
     void Zoom();
-    void RunScript();
     void SetPage();
     void LoadUrl(int times = 1);
+    void EmulationLevel();
+    void RunScriptWriteDOM();
```

Comment:
I think all these new methods should be in a single (`RunScript()`) method instead, there doesn't seem to be any advantage in splitting them and having 12 methods instead of 1 probably makes running the test noticeable slower (because the web view needs to be created and torn down 11 more times).